tests/parametric/test_trace_sampling.py

```diff
@@ -144,6 +145,51 @@ def test_trace_dropped_by_trace_sampling_rule(self, test_agent, test_library):
         assert span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == 0.0
 
 
+@features.trace_sampling
+@scenarios.parametric
+@rfc("https://docs.google.com/document/d/1S9pufnJjrsxH6pRbpigdYFwA5JjSdZ6iLZ-9E7PoAic/")
+class Test_Trace_Sampling_Globs_Feb2024_Revision:
```

Comment:
I advocate using another name instead of using dates. For example, the title of the rfc document

Reply:
The issue here is that there were updates to the RFC as it was finalized.
Some tracers already implemented the first set, so I chose to split the tests into different sets of revisions.